app/controllers/turfs_controller.rb

```diff
@@ -0,0 +1,23 @@
+class TurfsController < ApplicationController
+  def create
+    unless current_user
+      flash[:notice] = "You must be signed in to do that."
+      render "/"
+      return
+    end
+    params["lists"].each_value do |list|
+      @list = List.new
+      @list.name = list["name"]
+      if @list.save
+        list["voter_ids"].each do |voter_id|
```

Comment:
you're assuming a certain data structure in the params hash. You can't really assume the params hash will always be submitted in the way you intend it to. Your code should be defensive so it properly takes this into account.